lib/Files/TemplateLoader.php

```diff
@@ -33,7 +33,9 @@ class TemplateLoader {
 
 	public static function register(EventDispatcherInterface $dispatcher): void {
 		$dispatcher->addListener('OCA\Files::loadAdditionalScripts', function() {
-			self::loadTalkSidebarForFilesApp();
+			if (\OC::$server->getConfig()->getAppValue('spreed', 'conversations_files', '1') === '1') {
+				self::loadTalkSidebarForFilesApp();
+			}
```

Comment:
I would always call `self::loadTalkSidebarForFilesApp()` and then return without adding the style and script inside the function if the setting is not enabled, like done in [the TemplateLoader for the public share auth](https://github.com/nextcloud/spreed/blob/d4fc586f1a796b8cb19cff543b668470556e44b2/lib/PublicShareAuth/TemplateLoader.php).